core/src/main/java/io/grpc/internal/CensusTracingModule.java

```diff
@@ -211,9 +211,10 @@ private static EndSpanOptions createEndSpanOptions(
   }
 
   private static void recordNetworkEvent(
-      Span span, NetworkEvent.Type type,
+      Span span, io.opencensus.trace.NetworkEvent.Type type,
```

Comment:
Is it not the time yet to switch to the new class MessageEvent?

Reply:
I would prefer to do that in a separate PR since it's unrelated to RPC constants. Also, I assume we will still need to record for both `NetworkEvent` and `MessageEvent` to be backwards compatible? @bogdandrutu @hailongwen

Reply:
@sebright, do you mean old versions of opencensus-java implementations? So presumably, the new Census impl will treat NetworkEvent and MessageEvent as equivalent, then gRPC can switch to MessageEvent after all your customers has upgraded to the new impl.

Reply:
> @sebright, do you mean old versions of opencensus-java implementations?

I meant old versions of opencensus-api.  I don't think that the version of the OpenCensus implementation should matter to gRPC, as long as it is also compatible with the version of opencensus-api that gRPC uses.

The opencensus-java implementations still only reference NetworkEvent, and the API does the conversion.